back_translation/Backtranslate Using Google API.ipynb

```diff
@@ -0,0 +1,292 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    end_index = 50 #len(sentences)
</pre><p>Makes sense to have the provision to process a small set. Maybe for neatness we could have a variable, something like TEST_WITH_SMALL_SAMPLE = True/False? Then if true you could also change the filename to end with "....-data-sample.txt" to avoid any confusion.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SunbirdAI/parallel-text-EDA/pull/15/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
Done